### ISSUE TITLE
Can't load static assets that are inside folders

### ISSUE THREAD

**Issue by llorenspujol** [aws-amplify/amplify-console#1085]
### Description

I have a AWS Amplify manually deployed application.
The issue is, that I **can't load** static assets that are on folders, for example:

- `mydomain.amplifyapp.com/assets/i18n/en.json`
- `mydomain.amplifyapp.com/assets/images/pet.png`

Moving these assets to the root, like: `mydomain.amplifyapp.com/en.json`, solves the issue, but is not what I want.

### Current behavior

Digging more into the problem, the request: `mydomain.amplifyapp.com/assets/i18n/en.json` return a 301 code (redirect) with a redirect value to: `/assets/i18n/en.json/`, so the same address with an slash at the end:

![image](https://user-images.githubusercontent.com/18361241/94283038-cf621680-ff50-11ea-8c0f-a55e95aac1fe.png)

Obviously this isn't a correct path, so afterwards it gets redirected to /index.html because I have a custom rule that does so (the default rule for SPA).

### Expected behaviour

I expect any asset deployed to be loaded correctly (response 200 OK) independently if it is inside a folder or at the root.

### Comments

I could provide more detailed information, or a small reproduction of the project if needed.

Thanks in advance!

**Comment by litwicki** [comment]
@llorenspujol Is this a SPA? If so you'll need to add a rewrite rule for static assets if you're using something like react-router

**Comment by llorenspujol** [comment]
hi @litwicki ! yes is an SPA. Yes I had the rule for static assets and all going on. I also found the issue, so I am gonna close the ticket (I didn't remember to do it)

The issue was somehow related to the zip method I used to zip the app... Really strange since I tried on several machines to unzip it and the output was totally correct (environments: windows, ubuntu and linux). I don't know in which environment the unzip takes place on amplify, but for sure there is something **incompatible** with the node [cross-zip](https://github.com/feross/cross-zip) package.

The weird thing though is that unzip seems to works normally on amplify machine, but after that all the assets that are INSIDE folders return 301. 

The **solution** is just use another zipping method, in my case [**node-archiver**](https://github.com/archiverjs/node-archiver) works well.

I am closing the issue, just note that the **node package cross-zip doesn't work well with the unzip that amplify does**
